tools/serverpod_cli/test/integration/analyzer/dart/endpoint_validation/endpoint_ignore_annotation_test.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:io';
+
+import 'package:serverpod_cli/src/analyzer/dart/definitions.dart';
+import 'package:serverpod_cli/src/analyzer/dart/endpoints_analyzer.dart';
+import 'package:serverpod_cli/src/generator/code_generation_collector.dart';
+import 'package:serverpod_cli/src/test_util/endpoint_validation_helpers.dart';
+import 'package:serverpod_serialization/serverpod_serialization.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as path;
+
+const pathToServerpodRoot = '../../../../../../../..';
+var testProjectDirectory = Directory(path.joinAll([
+  'test',
+  'integration',
+  'analyzer',
+  'dart',
+  'endpoint_validation',
+  const Uuid().v4(),
+]));
+
+void main() {
+  setUpAll(() async {
+    await createTestEnvironment(testProjectDirectory, pathToServerpodRoot);
+  });
+
+  tearDownAll(() {
+    testProjectDirectory.deleteSync(recursive: true);
+  });
+
+  group('Given endpoint with IgnoreEndpoint annotation when analyzed', () {
```

Comment:
Suggestion: Add a test with some random annotation (that we should ignore) to validate we don't simply accept any annotation.